src/libgap.c

```diff
@@ -0,0 +1,219 @@
+/* LibGAP - a shared library version of the GAP kernel
+ * Copyright (C) 2013 Volker Braun <vbraun.name@gmail.com>
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version 2
+ * of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301,
+ * USA.
+ */
+
+
+#include "libgap.h"
+#include "system.h"
+#include "gap.h"
+#include "gasman.h"
+#include "libgap_internal.h"
+#include "scanner.h"
```

Comment:
The above will not work correctly in an HPC-GAP build. To make sure it always works right, you need to change `#include "foo.h"` to `#include <src/foo.h>`